backend/core/src/seeder/seeds/listings/listing-detroit-10136.ts

```diff
@@ -64,8 +64,8 @@ const listingSeed: ListingSeedType = {
   listingPreferences: [],
   jurisdictionName: "Detroit",
   marketingType: ListingMarketingTypeEnum.Marketing,
-  whatToExpect: `<div><strong>Vacant Units</strong>:<div className="mb-3">If you are looking to move in immediately, contact the property and ask if they have any vacant units.</div><div><strong>Waitlists</strong>:<div>If none are vacant, but you are still interested in living at the property in the future, ask how you can be placed on their waitlist.</div>`,
```

Comment:
Looking forward to our refactor of the seeds so we can make a default base that sets this once

Reply:
Since these are the same here and as defined on the partner side, is there a single place, and does it make sense, where we could define these and only have to change it in one place?